app/src/main/java/com/example/android/weatherapp/viewmodel/HomeScreenViewModel.kt

```diff
@@ -24,13 +27,15 @@ class HomeScreenViewModel : ViewModel() {
 
     private fun getCurrentWeather() {
         coroutineScope.launch {
-            var weatherObj = WeatherApi.retrofitService.getCurrentWeather()
+            var weatherObj = WeatherApi.retrofitService.getCurrentWeather("London", APP_ID)
             try {
```

Comment:
There are better ways to handle a Response object than try/catch. Try exploring documentation for Response.